packages/server/graphql/public/types/TeamPromptMeeting.ts

```diff
@@ -0,0 +1,27 @@
+import MeetingSeriesId from 'parabol-client/shared/gqlIds/MeetingSeriesId'
+import {TeamPromptMeetingResolvers} from '../resolverTypes'
+
+const TeamPromptMeeting: TeamPromptMeetingResolvers = {
+  meetingSeriesId: ({meetingSeriesId}, _args, _context) => {
+    if (meetingSeriesId) {
+      return MeetingSeriesId.join(meetingSeriesId)
+    }
+
+    return null
+  },
+  meetingSeries: async ({meetingSeriesId}, _args, {dataLoader}) => {
+    if (meetingSeriesId) {
+      const series = await dataLoader
+        .get('meetingSeries')
+        .load(MeetingSeriesId.join(meetingSeriesId))
```

Comment:
+1 preference to just pass in the database `meetingSeriesId` instead of the GraphQL GUID so we only have to do transforms for the ingress/egress, but if this is how the new dataloaders works i default to that pattern